### PR TITLE
Cleanup NMT notebook, fix image links

### PR DESCRIPTION
**Pull request by lamberta**
Stage: https://colab.sandbox.google.com/github/lamberta/tensorflow/blob/fix-nmt-nb/tensorflow/contrib/eager/python/examples/nmt_with_attention/nmt_with_attention.ipynb

### CODE REVIEW

**Comment by lamberta** [comment]
@yashk2810 PTAL

**Comment by yashk2810** [comment]
Looks good to me :)

File: tensorflow/contrib/eager/python/examples/nmt_with_attention/nmt_with_attention.ipynb [hunk 1]
```diff
@@ -224,9 +201,9 @@
       },
       "cell_type": "code",
       "source": [
-        "# first we remove the pronunciations\n",
-        "# second we clean the sentences\n",
-        "# and third we return word pairs in [ENGLISH, SPANISH] format\n",
+        "# 1. Remove the pronunciations\n",
```
Comment (MarkDaoust) [inline tensorflow/contrib/eager/python/examples/nmt_with_attention/nmt_with_attention.ipynb]: pronunciations? --> maybe punctuation? or accents?

**Review by MarkDaoust — DISMISSED** [review]
Changes LGTM.

File: tensorflow/contrib/eager/python/examples/nmt_with_attention/nmt_with_attention.ipynb (hunk 1 above)
Comment (yashk2810) [reply tensorflow/contrib/eager/python/examples/nmt_with_attention/nmt_with_attention.ipynb]: accents sounds more right

**Review by yashk2810 — APPROVED** [review]

**Review by MarkDaoust — APPROVED** [review]

**Pull request merged**